iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractTapChanger.java

```diff
@@ -117,19 +131,25 @@ public S getStep(int tapPosition) {
     }
 
     public S getCurrentStep() {
-        return getStep(getTapPosition());
+        int position = tapPosition.get(network.get().getVariantIndex());
+        if (position == Integer.MIN_VALUE) {
+            return null;
+        }
+        return getStep(position);
     }
 
     public boolean isRegulating() {
         return regulating.get(network.get().getVariantIndex());
```

Comment:
This boolean `regulating` is not part of the steady state hypotheses, as we have for generator voltage control for example?